suites/pacific/cephadm/tier-1_cephadm_cluster_view.yaml

```diff
@@ -0,0 +1,63 @@
+#===============================================================================================
```

Comment:
Instead of new test suite use exinting configs -
suite - https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephadm/tier-1_remove_cluster.yaml
conf - https://github.com/red-hat-storage/cephci/blob/master/conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml